automates/text_reading/src/test/scala/org/clulab/aske/automates/text/TestUnits.scala

```diff
@@ -4,10 +4,10 @@ import org.clulab.aske.automates.TestUtils._
 
 class TestUnits extends ExtractionTest {
 
-//todo: these tests are only here for the purpose of testing if the rules are extracting what they should; ultimately, they should be checking if the units are attached to variables themselves and not to the concepts/descriptions of variables
+  //todo: these tests are only here for the purpose of testing if the rules are extracting what they should; ultimately, they should be checking if the units are attached to variables themselves and not to the concepts/descriptions of variables
 
   val t1a = "The (average) daily net radiation expressed in megajoules per square metre per day (MJ m-2 day-1) is required."
-  passingTest should s"extract variables and units from t1a: ${t1a}" taggedAs(Somebody) in {
+  passingTest should s"extract variables and units from t1a: ${t1a}" taggedAs (Somebody) in {
```

Comment:
@alicekwak I think these `taggedAs(Somebody)` are function/method calls right? In the previous file they all seemed to be treated as method calls and did not have a space between the method name and the argument list. If these are method calls then the _best practice_ is to have them be `taggedAs(Somebody)` instead of `taggedAs (Somebody)`.

Can you please change the instances of `taggedAs (Somebody)` in this file back to `taggedAs(Somebody)`? Alternatively, if these are not method calls, then can you please explain to me what these constructs are so we can decided if we need to change the versions in all of our previous test files from `taggedAs(Somebody)` to `taggedAs (Somebody)`?

Reply:
I think it is a method call, so yes, no spaces. I also checked how they did it in Eidos and there is no space. The method is from the Scalatest package/library/whatever that is.

Reply:
Oops, I think that's my mistake. I'll make sure to delete the spaces.